apps/design/backend/src/ballot_styles.ts

```diff
@@ -1,42 +1,172 @@
-import { groupBy, throwIllegalValue, unique } from '@votingworks/basics';
+import {
+  groupBy,
+  throwIllegalValue,
+  unique,
+  typedAs,
+} from '@votingworks/basics';
 import {
   BallotLanguageConfigs,
   CandidateContest,
+  ContestOrderingSet,
   Contests,
   DistrictId,
+  Election,
   ElectionType,
+  OrderedCandidateContest,
+  OrderedYesNoContest,
   Parties,
-  PrecinctOrSplitId,
   Precinct,
+  PrecinctOrSplitId,
+  RotationParams,
   hasSplits,
 } from '@votingworks/types';
 import {
   generateBallotStyleGroupId,
   generateBallotStyleId,
 } from '@votingworks/utils';
 
+import {
+  BallotTemplateId,
+  getAllOrderedContestSetsForNhBallot,
+} from '@votingworks/hmpb';
 import { BallotStyle } from './types';
 
+/**
+ * Generates ordered contests for VxDefault ballot template.
+ * This template does not apply any rotation.
+ * Returns a single set with the contests in their original order.
+ */
+function getOrderedContestsWithoutRotation({
```

Comment:
defined the ordering for the default template, could be in libs/hmpb colocated with that template but I thought "no rotation" might be a more general case multiple templates use